diff --git a/node_modules/react-native-document-picker/src/index.tsx b/node_modules/react-native-document-picker/src/index.tsx
index 2eb9c86..370f915 100644
--- a/node_modules/react-native-document-picker/src/index.tsx
+++ b/node_modules/react-native-document-picker/src/index.tsx
@@ -1,6 +1,6 @@
 import { Platform, NativeModules, ModalPropsIOS } from 'react-native'
 import invariant from 'invariant'
-import type { PlatformTypes, SupportedPlatforms } from './fileTypes'
+import { PlatformTypes, SupportedPlatforms } from './fileTypes'
 import { perPlatformTypes } from './fileTypes'
 
 export type DocumentPickerResponse = {
